.drone.yml

```diff
@@ -576,7 +576,7 @@ steps:
   - cd $WORKSPACE_DIR/go/src/github.com/gravitational/teleport
   - build.assets/build-fido2-macos.sh build
   - export PKG_CONFIG_PATH="$(build.assets/build-fido2-macos.sh pkg_config_path)"
-  - make clean release OS=$OS ARCH=$ARCH FIDO2=yes TOUCHID=yes LIBPCSCLITE=yes
```

Comment:
Why was `LIBPCSCLITE=yes` removed here and below? cc @jakule

Reply:
Please see https://github.com/gravitational/teleport/pull/15874 where this value was updated in `dronegen/mac.go` but was not updated in the `.drone.yml` file via `make dronegen`.

Reply:
@Joerger added this change here: https://github.com/gravitational/teleport/pull/15874/files#diff-5a952c8230e4b996fed9112933bb065ab4efbf132bf59ef83811e8c5860cdd51

But he forgot to run dronegen afterwards.

Reply:
Gotcha - @jakule @Joerger can you confirm this is intended change?

Reply:
Yes this is correct, sorry for the confusion.